integrations/os2sync/os2mo.py

```diff
@@ -253,15 +248,59 @@ def kle_to_orgunit(orgunit, kle):
                 contactfortasks.add(uuid)
 
     if len(tasks):
-        orgunit["Tasks"] = list(tasks)
+        orgunit["Tasks"] = list(sorted(tasks))
 
     if len(contactfortasks):
-        orgunit["ContactForTasks"] = list(contactfortasks)
+        orgunit["ContactForTasks"] = list(sorted(contactfortasks))
+
+
+def is_ignored(unit, settings):
+    """Determine if unit should be left out of transfer
+
+    Example:
+        >>> settings={
+        ... "OS2SYNC_IGNORED_UNIT_LEVELS": ["10","2"],
+        ... "OS2SYNC_IGNORED_UNIT_TYPES":['6','7']}
+        >>> unit={"org_unit_level":{"uuid":"1"}, "org_unit_type":{"uuid":"5"}}
+        >>> is_ignored(unit, settings)
+        False
+        >>> unit={"org_unit_level":{"uuid":"2"}, "org_unit_type":{"uuid":"5"}}
+        >>> is_ignored(unit, settings)
+        True
+        >>> unit={"org_unit_level":{"uuid":"1"}, "org_unit_type":{"uuid":"6"}}
+        >>> is_ignored(unit, settings)
+        True
+
+    Args:
+        unit: The organization unit to enrich with kle information.
+        settings: a dictionary
+
+    Returns:
+        Boolean
+    """
+
+    if unit.get(
+        "org_unit_level"
+    ) and unit["org_unit_level"]["uuid"] in settings[
+        "OS2SYNC_IGNORED_UNIT_LEVELS"
+    ]:
+        return True
+    if unit.get(
+        "org_unit_type"
+    ) and unit["org_unit_type"]["uuid"] in settings[
+        "OS2SYNC_IGNORED_UNIT_TYPES"
+    ]:
+        return True
+
+    return False
```

Comment:
```suggestion
    return (
        unit.get("org_unit_level") and unit["org_unit_level"]["uuid"] in settings[
            "OS2SYNC_IGNORED_UNIT_LEVELS"
        ]
    ) or (
        unit.get("org_unit_type") and unit["org_unit_type"]["uuid"] in settings[
            "OS2SYNC_IGNORED_UNIT_TYPES"
        ]
    )
```
Alternativ til control flowet.

Reply:
Mere i stil med hvad der er lavet i `integrations/os2sync/lcdb_os2mo.py`